### DISCUSSION TITLE
How to choose out of a list of current organizations?

### DISCUSSION THREAD

**Discussion by GerlofdeHaan** [alkem-io/alkemio#652] [Q&A] [answered]
In the profile settings it is only possible to create a new organization to be associated to. Sometimes you just want to select out of the list of current organizations?

![afbeelding](https://user-images.githubusercontent.com/102361398/171845685-0a30ea1a-0338-46e6-ae7b-e23c2f6ec0f1.png)

**Comment by techsmyth** [comment, marked as answer]
Hi Gerlof,

The user association with Organizations is at the moment only possible from the settings on the Organization, so assuming you are admin on those organizations you can add yourself. 

There is an epic related to managing the user-organziation relationship: https://github.com/alkem-io/alkemio/issues/466 - but this is not near the top of the backlog. It is a clear gap, so happy to hear feedback re this should be higher that other ones (doc mgmt / wiki etc). 

One other comment: only Hub admins at the moment have the ability to create new organizations, so it is a restricted privilege for now. 

Cheers, Neil

**Comment by GerlofdeHaan** [comment]
Hi Neil,

Thanks.  Your suggestion worked.

Gr. Gerlof
Van: Neil Smyth ***@***.***>
Verzonden: zaterdag 4 juni 2022 15:52
Aan: alkem-io/alkemio ***@***.***>
CC: Gerlof de Haan ***@***.***>; Author ***@***.***>
Onderwerp: Re: [alkem-io/alkemio] How to choose out of a list of current organizations? (Discussion #652)


Hi Gerlof,

The user association with Organizations is at the moment only possible from the settings on the Organization, so assuming you are admin on those organizations you can add yourself.

There is an epic related to managing the user-organziation relationship: #466<https://github.com/alkem-io/alkemio/issues/466> - but this is not near the top of the backlog. It is a clear gap, so happy to hear feedback re this should be higher that other ones (doc mgmt / wiki etc).

Cheers, Neil
